File: PY-02/5/segitiga.py
baris = int(input("Jumlah baris: "))

for i in range(baris):
    for j in range(i + 1):
        print("*", end='')
    print()

print()

for i in range(baris):
    for j in range(i):
        print(" ", end='')
    for j in range(baris, i, -1):
        print("*", end='')
    print()
print()
for i in range(baris):
    #spasi
    for j in range(i, baris):
        print(" ", end='')
    #bintang
    for j in range(i + 1):
        print("*", end='')
    print()
print()
for i in range(baris):
    for j in range(baris, i, -1):
        print('*', end='')
    print()

File: CPP-02/Session 7/segitiga.cpp
#include <iostream>

using namespace std;

int main () {
    /*
    *
    **
    ***
    ****
    *****
    */
    for(int baris = 0; baris < 5; baris++) {
        for(int bintang = 0; bintang <= baris; bintang++) {
            cout << "*";
        }
        cout << endl;
    }
    
   cout << endl << endl;
    for(int baris = 0; baris < 5; baris++) {
        for(int spasi = 0; spasi < baris; spasi++) {
            cout << " ";
        }
        for(int bintang = 5; bintang > baris; bintang--) {
            cout << "*";
        }
        cout << endl;
    }
    /*
        *
       **
      ***
     ****
    *****
    */
    
    /*
    *****
    ****
    ***
    **
    *
    */
    return 0;
}